pkg/storage/store_test.go

```diff
@@ -32,8 +33,7 @@ func init() {
 	}
 	fakeServiceManager = fakeModule.ServiceManager
 	config := NewConfigWithDefaults()
-	config.RedisHost = "redis"
-	config.RedisDB = 1
+	config.RedisHost = os.Getenv("STORAGE_REDIS_HOST")
```

Comment:
See [this comment](https://github.com/Azure/open-service-broker-azure/pull/369/files#r177846967) for context.